ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSemanticAnalyzer.java

```diff
@@ -82,6 +87,7 @@ Licensed to the Apache Software Foundation (ASF) under one
 
   ReplicationSemanticAnalyzer(QueryState queryState) throws SemanticException {
     super(queryState);
+    this.conf = new HiveConf(super.conf);
```

Comment:
can you add a comment saying that new config is required because command specific config entries are added to it, and we don't want it to affect other queries in the session ?

Reply:
done